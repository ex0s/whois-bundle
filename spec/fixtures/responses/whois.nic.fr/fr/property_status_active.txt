%%
%% This is the AFNIC Whois server.
%%
%% complete date format : DD/MM/YYYY
%% short date format    : DD/MM
%% version              : FRNIC-2.5
%%
%% Rights restricted by copyright.
%% See http://www.afnic.fr/afnic/web/mentions-legales-whois_en
%%
%% Use '-h' option to obtain more information about this service.
%%
%% [80.181.162.148 REQUEST] >> google.fr
%%
%% RL Net [##########] - RL IP [#########.]
%%

domain:      google.fr
status:      ACTIVE
hold:        NO
holder-c:    GI658-FRNIC
admin-c:     TT599-FRNIC
tech-c:      MC239-FRNIC
zone-c:      NFC1-FRNIC
nsl-id:      NSL4386-FRNIC
registrar:   eMARKMONITOR Inc. dba MARKMONITOR
anniversary: 03/06
created:     27/07/2000
last-update: 03/06/2009
source:      FRNIC

ns-list:     NSL4386-FRNIC
nserver:     ns1.google.com
nserver:     ns2.google.com
nserver:     ns3.google.com
nserver:     ns4.google.com
source:      FRNIC

registrar:   eMARKMONITOR Inc. dba MARKMONITOR
type:        Isp Option 1
address:     Emerald Tech Center
address:     391 N. Ancestor Place
address:     ID 83704 BOISE
country:     US
phone:       +1 208 389 5740
fax-no:      +1 208 389 5771
e-mail:      ccops@markmonitor.com
website:     http://www.markmonitor.com
anonymous:   NO
registered:  10/01/2002
source:      FRNIC

nic-hdl:     GI658-FRNIC
type:        ORGANIZATION
contact:     Google Inc.
address:     1600, Amphitheatre Parkway
address:     94043 Mountain View Ca
country:     US
phone:       +1 650 253 0000
fax-no:      +1 650 618 8571
e-mail:      dns-admin@google.com
changed:     09/07/2009 nic@nic.fr
anonymous:   NO
obsoleted:   NO
idstatus:    dy
source:      FRNIC

nic-hdl:     TT599-FRNIC
type:        PERSON
contact:     Tu Tsao
address:     Google France
address:     38, avenue de l'Opera
address:     75002 Paris
country:     FR
phone:       +33 6 50 33 00 10
e-mail:      dns-admin@google.com
changed:     24/02/2009 ccops@markmonitor.com
anonymous:   NO
obsoleted:   NO
source:      FRNIC

nic-hdl:     MC239-FRNIC
type:        ROLE
contact:     MARKMONITOR CCOPS
address:     eMarkmonitor Inc. dba MarkMonitor
address:     PMB 155
address:     10400 Overland Road
address:     83709-1433 Boise, Id
address:     US
phone:       +01 2083895740
e-mail:      ccops@markmonitor.com
admin-c:     DL534-FRNIC
tech-c:      DL534-FRNIC
changed:     10/10/2008 ccops@markmonitor.com
anonymous:   NO
obsoleted:   NO
source:      FRNIC

