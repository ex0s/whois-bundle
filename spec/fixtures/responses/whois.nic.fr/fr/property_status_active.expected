#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true
